### PR TITLE
Add guide about Django settings

### PR DESCRIPTION
**Pull request by willbarton**
This PR adds a guide to this repo based on the discussion in #1 of a general standard for Django settings. It attempts to provide guidance and examples while being general enough that projects can have some flexibility as needed.

[Read it with GitHub markdown formatting in my fork](https://github.com/willbarton/back-end/blob/django-settings/django_settings.md).

@cfpb/back-end-team-admin

### CODE REVIEW

File: django_settings.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+```
+DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.mysql',
+            'NAME': {{ db_name }},
+            'USER': {{ db_user }},
+            'PASSWORD': {{ db_password }},
+            'HOST': {{ db_host }},  
+            'PORT': {{ db_port }},  
+        },
```
Comment (mehtadev17) [inline django_settings.md:83]: Add in something about SSL and other Postgres settings? 

```
### SSL Required in Prod ###
if DEBUG == False:
    DATABASES['DEFAULT']['OPTIONS'] = { 'sslmode': 'require', }
```

Or

```
'DEFAULT' : {
    'OPTIONS' : { 
        'sslmode': {{ db_sslmode }}, 
    }
```

Comment (willbarton) [inline django_settings.md:83]: I think that's a very good suggestion.

Comment (m3brown) [inline django_settings.md:83]: I believe that's just for postgres.  Unfortunantely mysql looks like this:

``` python
'OPTIONS': {
        'SSL': '/path/to/cert.pem',
    }
```

Comment (willbarton) [inline django_settings.md:83]:
```
'OPTIONS': {
        'SSL': {{ db_ssl_keypath }},
    }
```

Or if we want to to be database agnostic:

```
'OPTIONS': {
        {{ db_ssl_options }}
    }
```

:grin:

**Pull request closed without merging**